CMakeLists.txt

```
@@ -92,13 +92,23 @@ add_custom_command(
 file(RELATIVE_PATH DOXYGEN_BUILD_DIR_RELATIVE "${SOURCE_DOCS_DIR}" "${DOXYGEN_BUILD_DIR_CMAKE}")
 message(STATUS "DOXYGEN_BUILD_DIR_RELATIVE: ${DOXYGEN_BUILD_DIR_RELATIVE}")
 
+if(CMAKE_HOST_WIN32)
+    # On Windows, we have to build the documentation using only a single thread to
+    # avoid the build mysteriously taking forever.
+    # See https://github.com/LizardByte/Sunshine/pull/3361#issuecomment-2453549801.
```

```suggestion
    # See https://github.com/doxygen/doxygen/issues/9694
```

Probably better to just link to the upstream issue?